specification/components/schemas/responses/retrieveAttachment/422Response.yaml

```diff
@@ -0,0 +1,19 @@
+description: |
+  Where status code 422 (Unprocessable Entity) is returned then an eRS-OperationOutcome-1 will be included in the body, as detailed below. 
+  Check diagnostics property for specific information regarding the error.
+  
+    | Error code                         | Description                                                                                                                  |
+    | ---------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+    | REFERENCE_NOT_FOUND                | A supplied reference could not be resolved to valid resource (e.g. a patient, clinician or an organisation).                 |
+    | PATIENT_ERROR                      | An error occurred while retrieving the requested patient. Do not attempt again.                                              |
+    | INVALID_REQUEST_STATE              | The Request/Appointment Request, while otherwise valid, is not in the correct state for the specific action being performed. |
+    | NO_RELATIONSHIP                    | The user does not have a legitimate relationship with the referral.                                                          |
+headers:
```

Comment:
Are these descriptions correct for get Attachment or are we using these generic ones for all endpoints?

Reply:
Yeah these are the generic ones, I do realise they don't quite capture the logic for get Attachments so maybe we should define a more specific ones for this endpoint. I'll double check with Adam